clang/lib/Basic/Targets/SystemZ.h

```diff
@@ -91,11 +91,28 @@ class LLVM_LIBRARY_VISIBILITY SystemZTargetInfo : public TargetInfo {
                       "-v128:64-a:8:16-n32:64");
     }
     MaxAtomicPromoteWidth = MaxAtomicInlineWidth = 128;
+
+    // True if the backend supports operations on the half LLVM IR type.
+    // By setting this to false, conversions will happen for _Float16 around
+    // a statement by default, with operations done in float. However, if
+    // -ffloat16-excess-precision=none is given, no conversions will be made
+    // and instead the backend will promote each half operation to float
+    // individually.
+    HasLegalHalfType = false;
+    // Allow half arguments and return values (__fp16).
+    HalfArgsAndReturns = true;
```

Comment:
I don't think we should support this, this looks like a ARM-only feature.  It's false on Intel as well.

Reply:
ok removed. Interesting that this is left out on purpose even though it would be simple to allow...